docs/source-2.0/spec/documentation-traits.rst

```diff
@@ -139,10 +139,16 @@ Each ``example`` trait value is a structure with the following members:
       - :ref:`examples-ErrorExample-structure`
       - Provides an error shape ID and example error parameters for the
         operation.
-
-The values provided for the ``input`` and ``output`` members MUST be
-compatible with the shapes and constraints of the corresponding structure.
-These values use the same semantics and format as
+    * - disableConstraints
```

Comment:
Shouldn't this only be allowed for errors?

Reply:
ExamplesTraitsValidator checks that this is only set when errors are also present. Updated the docs to make this clear.